lib/net.js

```diff
@@ -980,9 +980,7 @@ Socket.prototype.connect = function(...args) {
   this.writable = true;
 
   if (pipe) {
-    if (typeof path !== 'string') {
-      throw new ERR_INVALID_ARG_TYPE('options.path', 'string', path);
-    }
+    validateString(path, 'path');
```

Comment:
Name changed from "options.path" to "path". I'm on mobile so can't check which one is correct.

Reply:
Technically both are valid.

Reply:
I think changing the name was a mistake, but going to put this back to `options.path` instead of `path` to make sure this is definitely not semver-major :)